d3a_api_client/utils.py

```diff
@@ -369,3 +370,12 @@ def read_simulation_config_file(config_file_path):
 def get_sim_id_and_domain_names():
     return simulation_id_from_env(), domain_name_from_env(), websocket_domain_name_from_env()
 
+
+def validate_client_up_to_date(response):
+    if "Api-Version" in response.headers:
+        remote_version = response.headers.get("Api-Version")
```

Comment:
Nit: since you've already checked for the existance of the `"Api-Version"` key, in the second like you could also omit the `.get()` and just use `["Api-Version"]`. Or it could become something like 
```suggestion
remote_version = response.headers.get("Api-Version")
if not remote_version:
    return

if __version__ < remote_version:
    ...
```